src/api/route-services/samples.js

```diff
@@ -192,9 +205,31 @@ class SamplesService {
       await dynamodb.updateItem(params).promise();
     };
 
+    const addSampleIdToExperiments = async () => {
+      const dynamodb = createDynamoDbInstance();
+
+      const marshalledKey = convertToDynamoDbRecord({
+        experimentId,
+      });
+
+      const marshalledData = convertToDynamoDbRecord({
+        ':newSampleId': [newSample.uuid],
+      });
+
+      const params = {
+        TableName: this.experimentsTableName,
+        Key: marshalledKey,
+        UpdateExpression: 'SET sampleIds = list_append(sampleIds, :newSampleId)',
+        ExpressionAttributeValues: marshalledData,
+      };
+
+      await dynamodb.updateItem(params).promise();
+    };
+
```

Comment:
I am surprised that we have to write this piece of code from scratch -- where in the api do we handle adding sample ids in the experiments table?

Reply:
We don't. What we do to insert new sample ids to experiment is insert it into `experiments.sampleIds` redux and then save the whole updated `experiments` object.